conftest.py

```diff
@@ -70,16 +70,83 @@ def flask_client(app):
     return app.test_client()
 
 
+@pytest.fixture
+def cli_runner(app):
+    return app.test_cli_runner()
+
+
 @pytest.fixture
 def dummy_uid():
     return 'user123'
 
 
+@pytest.fixture
+def dummy_uids():
+    return ['user124', 'user125', 'user126']
+
+
 @pytest.fixture
 def dummy_participant_uid():
     return 'pig'
 
 
+@pytest.fixture
+def create_newdle(dummy_uid, db_session):
+    """Returns a callable which lets you create dummy newdles"""
+
+    def _create_newdle(id_=None, **kwargs):
```

Comment:
i'm tempted to just use `id` and not bother about shadowing the builtin... it's not like we use this in here anyway